aws_lambda_builders/workflows/dotnet_clipackage/workflow.py

```diff
@@ -0,0 +1,46 @@
+"""
+.NET Core CLI Package Workflow
+"""
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+
+from .actions import GlobalToolInstallAction, RunPackageAction
+from .dotnetcli import SubprocessDotnetCLI
+
+
+
+class DotnetCliPackageWorkflow(BaseWorkflow):
+
+    """
+    A Lambda builder workflow that knows to build and package .NET Core Lambda functions
+    """
+    NAME = "DotnetCliPackageBuilder"
+
+    CAPABILITY = Capability(language="dotnet",
```

Comment:
Do we need to add resolver and validator for the executables used by the workflow?

Reply:
Do you have an example of what you mean?

Reply:
https://github.com/awslabs/aws-lambda-builders/blob/develop/aws_lambda_builders/workflows/java_gradle/workflow.py#L54

https://github.com/awslabs/aws-lambda-builders/blob/d60920060e2f52688a1b84d8c6d635dd8d4799fc/aws_lambda_builders/workflows/java_gradle/workflow.py#L57

Those are the examples.

Reply:
I added a resolver to make sure the dotnet CLI is installed. 

A validator is not needed because the dotnet CLI has its own built in versioning system to match the available installed .NET SDK with the selected project. A validator would just duplicate that work and possibly get in the way.